tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java

```diff
@@ -95,12 +103,48 @@ public Tablet(String deviceId, List<MeasurementSchema> schemas, int maxRowNumber
     this.deviceId = deviceId;
     this.schemas = schemas;
     this.maxRowNumber = maxRowNumber;
+    mschemawithindex=new HashMap<>();
+
+    for(int i=0;i<schemas.size();i++){
```

Comment:
There is a java-google-style.xml under the root folder, import it as the default code style and format the code.

It should be something like this:
for(int i = 0; i < schemas.size(); i++){

Reply:
copy that,thank you